aten/src/ATen/native/cudnn/Conv_v8.cpp

```diff
@@ -373,7 +373,7 @@ void generate_and_filter_plans(const cudnnHandle_t handle, cudnn_frontend::Opera
   if (remove_invalid) {
     cudnn_frontend::executionPlans_t new_valid_plans;
     for (auto &plan : valid_plans) {
-      if (plan.getWorkspaceSize() <= max_workspace_size) {
+      if ((size_t) plan.getWorkspaceSize() <= max_workspace_size) {
```

Comment:
Why this change is needed? If it is, then please use `static_cast<size_t>(plan.getWorkspaceSize())`
```suggestion
      if (plan.getWorkspaceSize() <= max_workspace_size) {
```

Reply:
That's our favorite signed-unsigned comparison probably, can we enable compilation flags to error on it in OSS builds? It's been a nuisance.

Reply:
Given that `getWorkspaceSize()` returns `int64_t` in cudnn frontend, a better fix would be to change our vars like max_worksplace_size and curr_workspace_size to int64_t instead of size_t

Reply:
sounds good! I'll make the change and see if I can make this from warning to error in oss :)